### ISSUE TITLE
Review permissions for editing additional resources for groups

### ISSUE THREAD

**Issue by rjsparks** [ietf-tools/datatracker#3665]
The permission logic that allows additional resources is likely lagging behind changes to how we are using groups. Chairs of directorates, for example, should be able to edit the additional resources for those groups, and chairs of EMODIR currently cannot.

**Comment by rjsparks** [comment]
@jennifer-richards have we already addressed this?

**Comment by jennifer-richards** [comment]
I don't think so. We did some data-driven permissions related to EMODIR for #4004, but that was for interim requests. I don't think we did anything on resource editing.
